### PR TITLE
Fix meta for aten.upsample_bilinear2d.vec

### PR DESCRIPTION
**Pull request by SherlockNoMad**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #88175
* __->__ #88158

### CODE REVIEW

File: torch/_decomp/decompositions.py [hunk 1]
```diff
@@ -1886,6 +1886,11 @@ def upsample_bilinear2d_vec(
     q1 = torch.mul(v1, xscale1) + torch.mul(v2, xscale2)
     q2 = torch.mul(v3, xscale1) + torch.mul(v4, xscale2)
     result = torch.mul(q1, yscale1) + torch.mul(q2, yscale2)
+
+    # convert output to correct memory format, if necessary
+    input_memory_format = utils.suggest_memory_format(input)
+    result = result.contiguous(memory_format=input_memory_format)
```
Comment (SherlockNoMad) [inline torch/_decomp/decompositions.py:1892]: I wonder if this has any difference from
`.to(memory_format=input_memory_format)`

What's the proper way to do this 

Alternatively, I can 
- "empty + fill_",  not ideal as it introduce a non-functional op 
- "empty + torch.fill" ?

Comment (ngimel) [reply torch/_decomp/decompositions.py:1892]: Yes, this is different from `to(memory_format=input_memory_format)`, `to` would do nothing for non-channels-last memory format. `contiguous` is good.

**Review by ngimel — APPROVED** [review]

**Pull request closed without merging**